docs/src/lib/transform.ts

```diff
@@ -70,6 +69,36 @@ function transformCommentDisplayPart(
   return functionNames.has(codeContent) ? `[${text}](#${codeContent})` : text;
 }
 
+function getMapping({ blockTags }: JSONOutput.Comment) {
+  const result = new Array<{
+    library: string;
+    name: string;
+    notes: string | undefined;
+  }>();
+
+  if (blockTags === undefined) {
+    return result;
+  }
+
+  const mappingTags = blockTags.filter(({ tag }) => tag === `@mapping`);
+
+  for (const { content } of mappingTags) {
+    const textContent = content.map(({ text }) => text).join("");
+
+    // Matches "<library> <name> [- notes]":
+    const match = textContent.match(/(\w+) (\w+)(?: - (.+))?/);
+    const [, library, name, notes] = match ?? [];
+
+    if (!library || !name) {
```

Comment:
This should probably throw so we fail the build on bad doc strings.
also, don't use implicit boolean checks, prefer !== undefined, or !== "", or whatever library and name are here (I don't know the type for regex matches).